mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -15,7 +15,7 @@
 
 
 """A tunable parameter value type alias."""
-TunableValue = Union[int, float, str]
+TunableValue = Union[int, float, Optional[str]]
```

Comment:
Need to be able to support `null` in the json.

Reply:
Maybe a bit more elegant would be to write `Union[int, float, str, None]` - btw that would directly translate into Python 3.10 `int | float | str | None`